extensions/rich_text_components/base.py

```diff
@@ -49,6 +49,12 @@ class BaseRichTextComponent(object):
     # Whether the component requires the filesystem in some way that
     # prevents it from being used by unauthorized users.
     requires_fs = False
+    # Whether the component should be displayed as a block component
+    is_block = False
```

Comment:
is_block_element is a better name, I think -- one says that a div is a "block element", not a "block".